cmd/thanos/bucket.go

```diff
@@ -582,10 +582,21 @@ func (t Table) Less(i, j int) bool {
 }
 
 func compare(s1, s2 string) bool {
+	// Values can be either Time, Duration or other.
 	s1Time, s1Err := time.Parse("02-01-2006 15:04:05", s1)
 	s2Time, s2Err := time.Parse("02-01-2006 15:04:05", s2)
 	if s1Err != nil || s2Err != nil {
-		return s1 < s2
+		s1Duration, s1Err := time.ParseDuration(s1)
+		s2Duration, s2Err := time.ParseDuration(s2)
+		if s1Err != nil || s2Err != nil {
+			s1Int, s1Err := strconv.Atoi(strings.Replace(s1, ",", "", -1))
```

Comment:
This will run into problems once that string won't fit into an `int`. We should opt to use:
```
strconv.ParseUint(strings.Replace(s1, ",", "", -1), 10, 64)
```
Which will give us an `uint64`. This is a real possibility because we use `uint64` underneath on:
* `blockMeta.Stats.NumSeries`
* `blockMeta.Stats.NumSamples`
* `blockMeta.Stats.NumChunks`